### DISCUSSION TITLE
No link to previews download in the doc

### DISCUSSION THREAD

**Discussion by matm** [eduardofilo/RG350_adam_image#183] [SimpleMenu topics]
Sorry if it has been asked hundred times but I didn't find the answer in the README. Where am I supposed to download the previews for all supported systems (into `/roms/$SYSTEM/.previews/`)?

Thank you

**Comment by eduardofilo** [comment]
Sorry, I don't understand well.
Are you asking **from where** can be the previews downloaded?

**Comment by JAF62** [comment]
I find Skraper app on the PC is the best way to do this.
Tutorial and instructions here: https://youtu.be/RBQuFBbRsXs

**Reply by matm** [reply]
Thanks! Looks like a great solution but for Windows only (for now). The Linux download link is the 7z distribution for Windows.

**Comment by matm** [comment]
@eduardofilo Sorry if I wasn't clear :) Yes, I have this Adam image working on my device but want to add the extra previews images for games. I have no idea where they are coming from. Thank you.

**Reply by eduardofilo** [reply]
Keep in mind that although skraper automates and facilitates the task, you can always create your previews manually. To do this you just have to put the external card in the PC as you do to transfer ROMS or BIOS and copy png files in the corresponding directory with the same file name except the extension.

For example, if you have a GBC game called `1942 (USA, Europe).gbc` (or `1942 (USA, Europe).zip` if you have it zipped) in the `<ext_sd>/roms/GB` directory, then the preview should be `<ext_sd>/roms/GB/.previews/1942 (USA, Europe).png`

**Reply by matm** [reply]
OK, thanks for the example @eduardofilo! Don't you think it may be worth adding some doc about it into the README. I can create an MR if you find it relevant.

**Reply by eduardofilo** [reply]
I will take your suggestion into account. In the meantime, if you want to advance some work, I'd appreciate it.

**Comment by eduardofilo** [comment]
OK. In that case, the best option is the one @JAF62 told you about. I have this guide but it is in Spanish. You may be able to use the browser's translation function.
http://apuntes.eduardofilo.es/2020-01-11-rg350_scraper.html

Although it doesn't bother me, because github is very extensive and it's not always easy to get around it, please, in cases like these that don't involve an incident with the image, use the Discussions section instead of opening an issue.

Greetings.

**Reply by neilswann80** [reply]
RetroGameCorps has a great guide on Skraper too:
https://retrogamecorps.com/2021/04/02/quick-guide-skraper-for-retro-handheld-devices/
